src/condor_schedd.V6/schedd.cpp

```diff
@@ -13644,6 +13644,12 @@ Scheduler::Init()
 	JobStopCount = param_integer( "JOB_STOP_COUNT", 1, 1 );
 	stop_job_queue.setCountPerInterval( JobStopCount );
 
+	m_protected_url_map.clear();
+	std::string protectedUrlMapFile;
+	param(protectedUrlMapFile, "PROTECTED_URL_TRANSFER_MAPFILE");
```

Comment:
param into std:;string makes a copy of the value,  and adds code.  
`auto_free_ptr foo(param("key"));  if (foo) function(foo.ptr())` is cleaner *and* more efficient.